pkgs/applications/editors/vim/plugins/nodePackagePlugins.nix

```diff
@@ -63,42 +58,3 @@ self: super:
   in
   lib.genAttrs nodePackageNames nodePackage2VimPackage
```

Comment:
Same, may as well inline `nodePackage2VimPackage` given this file is now dedicated to node packages, I don't think having a binding adds any readability anymore.